package.json

```diff
@@ -44,11 +44,13 @@
   "devDependencies": {
     "@types/chai": "3.4.22-alpha",
     "@types/mocha": "2.2.21-alpha",
+    "@types/mock-fs": "3.6.22-alpha",
```

Comment:
Unfortunately, typings aren't always up to date, but event the older ones are usually good enough.